### ISSUE TITLE
Look for a profit model for neuron

### ISSUE THREAD

**Issue by Keith-CY** [Magickbase/neuron-public-issues#292]
I've been considering the profit model for Neuron recently.

As we know, Neuron software is an open-source wallet developed with security as the top priority. Therefore, unnecessary features should not be added, so it's currently challenging to include value-added services. Additionally, given Neuron's positioning as a desktop full-node wallet, it's difficult to attract consumer end-users.

Hence, I'd like to explore a model that generates revenue by providing value-added services through an extension of the Neuron software.

Neuron, as a security-first full-node wallet, has two major needs:
1. Addressing synchronization efficiency and stability issues.
2. Enhancing transaction scalability.

Neuron's positioning introduces both of these issues.

Regarding point 1, synchronization efficiency and stability, even with various optimizations at the client level, we can only make slight improvements in the user experience, which cannot match wallets with server support. Recent requests show that many users want to use Neuron as a cold wallet and subscribe to assets through other online wallets. We can offer an online watching wallet service that supports Neuron as an entry point, providing value-added services through an online platform.

As for point 2, transaction scalability is limited by Neuron's nature and positioning. Since Neuron is a desktop application that does not support hot updates, it cannot quickly iterate on detailed feature improvements. Releasing new features requires a lengthy polishing process and cannot achieve rapid iterative updates. Moreover, considering that many applications/standards are not yet stable, Neuron should not support them prematurely. Therefore, an online service on the web can compensate for Neuron's lack of functionality.

So, my idea is to use Neuron software as the core component and extend its performance and functionality through the web, thereby expanding the user base and gradually introducing paid features.

**Comment by homura** [comment]
So does it mean that Neuron could be a fully offline cold wallet and delegate sync to a SaaS? The functions like transferring, exploring (CKB, NFT, UDT), and Nervos DAO can be delegated to the SaaS, and the offline Neuron only be in charge of signing.

**Comment by Keith-CY** [comment]
> So does it mean that Neuron could be a fully offline cold wallet and delegate sync to a SaaS? The functions like transferring, exploring (CKB, NFT, UDT), and Nervos DAO can be delegated to the SaaS, and the offline Neuron is only in charge of the signing.

Indeed, this idea came long ago, but was halted due to historical reasons. It's mentioned again because I think a healthy profit model does matter, and should be scheduled in advance.

**Comment by yanguoyu** [comment]
For the first idea, we can create an online wallet with the wallet protocol.

**Comment by Keith-CY** [comment]
> For the first idea, we can create an online wallet with the wallet protocol.

Sure, the account-level asset page on CKB explorer is for this kind of watching wallet.

Furthermore, we can extend the Neuron project page to be a self-host watching wallet that runs in a browser. It can be done by loading a local XPub key and running a local lumos in the browser. It's decentralized and easy to deliver.

**Comment by Keith-CY** [comment]
Here's how Liana offers different plans
Ref: https://wizardsardine.com/liana/#plans
